developers/subgraph/get-veocean-stats.md

```diff
@@ -4,7 +4,7 @@ description: 'Discover the World of veOCEAN: Retrieving a Stats'
 
 # Get veOCEAN stats
 
-If you are already familiarized with veOCEAN, you're off to a great start. However, if you need a refresher, we recommend visiting the [veOCEAN](../../rewards/veocean.md) page for a quick overview :mag:
+If you are already familiarized with veOCEAN, you're off to a great start. However, if you need a refresher, we recommend visiting the [veOCEAN](../../rewards/df-veocean.md) page for a quick overview :mag:
```

Comment:
Hm, even if this links points to a file with the right path, it doesn't seem to work on the preview generated. Let's revalidate it once this PR is merged

Reply:
It works fine locally